platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -252,6 +254,13 @@ public static PlatformSchedulers create(
                         .withUnhandledTaskCapacity(config.issDetectorUnhandledCapacity())
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
+                        .cast(),
+                model.schedulerBuilder("hashLogger")
+                        .withType(TaskSchedulerType.SEQUENTIAL_THREAD)
+                        .withUnhandledTaskCapacity(config.hashLoggerUnhandledTaskCapacity())
+                        .withOnRamp(hashingObjectCounter)
```

Comment:
```suggestion
```
This is not necessary, and will probably break event hashing back pressure.

In general, calling `withOnRamp()` or `withOffRamp()` is not required except in extremely rare edge cases where we need to do something complicated with back pressure (event hashing is currently the only such edge case). Nuanced topic, let me know if you'd like to go more in depth over a a whiteboard.

Reply:
removed